roofit/roofit/src/RooBernstein.cxx

```diff
@@ -211,11 +228,10 @@ RooSpan<double> RooBernstein::evaluateBatch(std::size_t begin, std::size_t batch
 
 Int_t RooBernstein::getAnalyticalIntegral(RooArgSet& allVars, RooArgSet& analVars, const char* rangeName) const
 {
-  if (rangeName && strlen(rangeName)) {
-    return 0 ;
-  }
+  if (rangeName && strlen(rangeName))   return 0;
```

Comment:
Is it true that a Bernstein cannot be integrated over a range? I don't think so. Would it be possible to fix this defect of the old implementation?